jsapp/js/components/processing/sidebar/sidebarLabelsSettings.component.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from 'react';
+import Button from 'js/components/common/button';
+import KoboModal from 'js/components/modals/koboModal';
+import KoboModalHeader from 'js/components/modals/koboModalHeader';
+import KoboModalFooter from 'js/components/modals/koboModalFooter';
+import KoboSelect from 'js/components/common/koboSelect';
+import singleProcessingStore from 'js/components/processing/singleProcessingStore';
+
+import styles from './sidebarLabelsSettings.module.scss';
+import {LanguageCode} from '../../languages/languagesStore';
+
+export default function SidebarLabelsSettings() {
+  const [store] = useState(() => singleProcessingStore);
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+
+  const [labelLanguage, setLabelLanguage] = useState<LanguageCode | string>(
+    store.getCurrentlyDisplayedLanguage()
+  );
+
+  const displayedLanguageList = store.getDisplayedLanguagesList();
+
+  return (
+    <div className={styles.root}>
+      <Button
+        size='m'
+        type='bare'
+        label={t('Change question language')}
+        color='storm'
+        startIcon='globe-alt'
+        onClick={() => setIsModalOpen(true)}
+      />
+      <KoboModal
+        isOpen={isModalOpen}
+        onRequestClose={() => {
+          // Reset modals and checkboxes if user closed modal without applying
+          setIsModalOpen(false);
+        }}
+        size='medium'
+      >
+        <KoboModalHeader>{t('Change label language')}</KoboModalHeader>
```

Comment:
Maybe "Change labels display language"? Or we could copy this 1-to-1:
<img width="245" alt="Screenshot 2024-06-11 at 15 08 48" src="https://github.com/kobotoolbox/kpi/assets/2521888/fa64bdba-14f5-4659-83ee-e5bafc9c7038">
Because you know "XML Values" is not a "language" per se, so you're not really "selecting a language" when you choose it ;-)